.github/workflows/deploy.yml

```
@@ -0,0 +1,120 @@
+        uses: cloudflare/pages-action@v1
+        with:
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          projectName: blue-badge-pwa-staging
+          directory: dist
+          gitHubToken: ${{ secrets.GITHUB_TOKEN }}
+
+  test-pwa:
+    runs-on: ubuntu-latest
+    needs: build-and-deploy
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)